### PR TITLE
Update endpoint-dlp-learn-about.md

### PR DESCRIPTION
**Pull request by MasLibman**
Fixing Windows 10/11
@chrfox

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/endpoint-dlp-learn-about.md [hunk 1]
```diff
@@ -25,14 +25,14 @@ description: "Microsoft 365 Endpoint data loss prevention extends monitoring of
...
 
 > [!TIP]
 > If you are looking for device control for removable storage, see [Microsoft Defender for Endpoint Device Control Removable Storage Access Control](../security/defender-endpoint/device-control-removable-storage-access-control.md#microsoft-defender-for-endpoint-device-control-removable-storage-access-control).
 
 ## Endpoint activities you can monitor and take action on
 
-Microsoft Endpoint DLP enables you to audit and manage the following types of activities users take on sensitive items that are physically stored Windows 10 devices or macOS devices.
+Microsoft Endpoint DLP enables you to audit and manage the following types of activities users take on sensitive items that are physically stored Windows 10, Windows 11, or macOS devices.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-learn-about.md:35]:
```suggestion
Microsoft Endpoint DLP enables you to audit and manage the following types of activities users take on sensitive items that are physically stored on Windows 10, Windows 11, or macOS devices.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MasLibman, kindly commit the minor additional suggestion. Many thanks.

**Comment by yogkumgit** [comment]
@MasLibman Please update the changes.

**Pull request merged**